plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/InlineStmtConversionContext.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.conversion
+
+import org.jetbrains.kotlin.formver.embeddings.TypeEmbedding
+import org.jetbrains.kotlin.formver.embeddings.VariableEmbedding
+import org.jetbrains.kotlin.formver.viper.MangledName
+
+class InlineStmtConversionContext(
+    private val methodCtx: MethodConversionContext,
+    private val substitutionParams: Map<MangledName, MangledName>,
```

Comment:
What is the role of these params? I think a comment explaining what they represent would at least be good.